src/themes/dspace/app/header/header.component.html

```diff
@@ -17,34 +17,34 @@
             <div class="lindat-block lindat-block--clariah-theme-main-menu">
               <ul class="lindat-nav lindat-navbar-nav">
                 <li class="lindat-nav-item ">
-                  <a routerLink="community-list" class="lindat-nav-link">{{'navbar.community-list' | translate}}</a>
+                  <a href="services/catalog" class="lindat-nav-link">{{'navbar.community-list' | translate}}</a>
                 </li>
                 <li class="lindat-nav-item ">
                   <a routerLink="home" class="lindat-nav-link">{{'navbar.repository' | translate}}</a>
                 </li>
                 <li class="lindat-nav-item ">
-                  <a routerLink="education" class="lindat-nav-link">{{'navbar.education' | translate}}</a>
+                  <a href="#education" class="lindat-nav-link">{{'navbar.education' | translate}}</a>
                 </li>
                 <li class="lindat-nav-item ">
-                  <a routerLink="projects" class="lindat-nav-link">{{'navbar.project' | translate}}</a>
+                  <a href="#projects" class="lindat-nav-link">{{'navbar.project' | translate}}</a>
                 </li>
                 <li class="lindat-nav-item ">
-                  <a routerLink="tools" class="lindat-nav-link ">{{'navbar.tools' | translate}}</a>
+                  <a href="#tools" class="lindat-nav-link ">{{'navbar.tools' | translate}}</a>
                 </li>
                 <li class="lindat-nav-item ">
-                  <a routerLink="en/services" class="lindat-nav-link ">{{'navbar.services' | translate}}</a>
+                  <a href="en/services" class="lindat-nav-link ">{{'navbar.services' | translate}}</a>
```

Comment:
When you switch to Czech language there is `cs/services`. Use some `localeService` - find how it is used somewhere.
`cs` path in the url is also used in the other redirects.